examples/type-cast/type_cast.out

```diff
@@ -0,0 +1,12 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file and issue the `ballerina run` command.
```

Comment:
```suggestion
# `.bal` file and execute the `ballerina run` command.
```

Reply:
"issue" is what we've used in all the BBEs. If "execute" is more accurate shall we fix all the BBEs separately?

Reply:
It seems like people are using both "issue" and "execute" words for the same purpose. It seems like, The word "execute" is used more frequently on the web in the context of computing.

Reply:
Yeah, "execute" or even "use" seems natural with the usage of a command compared to "issue". So I'll update this here, and send a separate PR to update the rest of the BBEs.